src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,79 @@
+package homework_2.random_chars_table;
+
+import java.util.*;
+
+public class RandomCharsTable {
+
+    protected static final String ANSI_RED = "\u001B[31m";
+    protected static final String ANSI_RESET = "\033[0m";
+
+    public void start() {
+        Scanner systemScanner = new Scanner(System.in);
+        Scanner stringScanner;
+        char[][] chars;
+        String parity;
+        List<Character> characterList;
```

Comment:
Most of these can be inside if block. No need to create them before you need them